components/atom/validationCode/src/index.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import {useState, useRef, createRef} from 'react'
+
+const DEFAULT_LENGTH = 6
+const BASE_CLASS = 'react-AtomValidationCode'
+const noop = () => {}
+
+export default function AtomValidationCode({
+  onChange = noop,
+  label,
+  length = DEFAULT_LENGTH
+}) {
+  const [numbers, setNumbers] = useState(new Array(length).fill(null))
+
+  const numberRefs = useRef([])
+  numberRefs.current = numbers.map(
+    (_, i) => numberRefs.current[i] ?? createRef()
+  )
+
+  const handleChange = ({i, value}) => {
+    if (value < 10) {
```

Comment:
Maybe we can assign this 10 into a variable so we can give it some meaning. Is it the MAX_NUMBER_FOR_EACH_VALUE maybe?